integration_test/gov_module/gov_proposal_test.yaml

```diff
@@ -0,0 +1,73 @@
+- name: Test making a new param change proposal should pass and take effect
```

Comment:
Can we also add integration tests for the following failure scenarios where the coins are burned?

1. https://github.com/sei-protocol/sei-cosmos/blob/main/x/gov/keeper/tally.go#LL98C1-L104C3 
2. https://github.com/sei-protocol/sei-cosmos/blob/main/x/gov/keeper/tally.go#L111-L114 

Perhaps also the failure scenarios where its not burned too

Reply:
Good idea!